dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -10,6 +12,6 @@ USER root
 # Install less for handling git diff properly
 # Install sshpass for handling passwords for SSH keys
 RUN yum install -y sudo git bzip2 which bash curl openssh less && \
-    wget https://netcologne.dl.sourceforge.net/project/sshpass/sshpass/1.06/sshpass-1.06.tar.gz -O sshpass.tar.gz && \
-    tar -xvf sshpass.tar.gz && cd sshpass-1.06 && ./configure && make install && cd .. && rm -rf sshpass-1.06 && \
+    wget https://netcologne.dl.sourceforge.net/project/sshpass/sshpass/"${SSHPASS_VERSION}"/sshpass-"${SSHPASS_VERSION}".tar.gz -O sshpass.tar.gz && \
```

Comment:
I notice you're using `wget` but you've installed `curl`. 

Why not use `curl -sSL` instead? 

Note: `-O` will use the remote filename (`sshpass-"${SSHPASS_VERSION}".tar.gz`, so if you want to rename it on the fly, use `-o sshpass.tar.gz `.

Reply:
changed to `curl`